cloudant-http/build.gradle

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+dependencies {
+    compile group: 'com.google.code.gson', name: 'gson', version: '2.2.4'
```

Comment:
As per https://github.com/cloudant/java-cloudant/pull/223#discussion_r55206793 I have split the dependencies into the correct sub-projects and removed them from the top level.